doc/api/http2.md

```diff
@@ -3673,9 +3673,8 @@ will be emitted.
 
 ```js
 const body = 'hello world';
-response.writeHead(200, {
-  'Content-Length': Buffer.byteLength(body),
-  'Content-Type': 'text/plain; charset=utf-8' });
+response.writeHead(200, { 'Content-Length': Buffer.byteLength(body),
```

Comment:
I'd rather solve these kinds of situations by moving the `});` to the next line instead of having to indent it all the way over to the right.

Reply:
I agree. Unfortunately this is the way it auto-fixes these things. I can go through and manually adjust them, though...tedious, but probably worth it to keep the code a bit more readable.